packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1485,3 +1334,15 @@ describe('TokenListController', () => {
     });
   });
 });
+
+/**
+ * Construct the path used to fetch tokens that we can pass to `nock`.
+ *
+ * @param chainId - The chain ID.
+ * @returns The constructed path.
+ */
+function getTokensPath(chainId: Hex) {
+  return `/tokens/${convertHexToDecimal(
+    chainId,
+  )}?occurrenceFloor=3&includeNativeAssets=false&includeDuplicateSymbolAssets=false&includeTokenFees=false&includeAssetType=false`;
```

Comment:
Nit: as we repeat this set of options passed to the URL in several assets controllers and tests, perhaps it might be worth declaring a constant and reusing it?

Reply:
Yes, that's true, I do end up repeating this. I suppose we could extract these options to a variable, but then they would be separate from the helper function that uses them, and I thought that would feel off. Maybe we can revisit if it becomes an issue.